crates/sui-indexer-alt-framework/src/ingestion/client.rs

```diff
@@ -214,3 +214,183 @@ impl IngestionClient {
         Ok(Arc::new(data))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use prometheus::Registry;
+    use std::collections::HashMap;
+    use std::sync::Arc;
+    use std::time::Duration;
+    use tokio::sync::Mutex;
```

Comment:
In case you do want to keep using `Mutex<HashMap<...>>` you probably want to use `std::sync::Mutex`, not `tokio::sync::Mutex` (you can read more about that [here](https://docs.rs/tokio/1.24.2/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use)).